src/UIAutomation/UiaOperationAbstraction/UiaTypeAbstractionImpl.g.cpp

```diff
@@ -7008,7 +7008,10 @@
             auto element = std::get<AutomationRemoteElement>(m_member).GetParentElement();
             if (cacheRequest)
             {
-                element.PopulateCache(*cacheRequest);
+                delegator->If(element.IsNull().BoolNot(), [&]()

```

Comment:
Good catch! I'm not 100% sure what you mean by "the remote operation is operated on the provider side", but yeah this would cause issues if we try to navigate to an out-of-proc parent I think.

Reply:
What I mean is that, if we don't add this check, UIA remote operation on the provider side will expect an element operand to be evaluated, but instead it will actually get a null operand, which eventually leads to an exception being thrown and the whole operation abort due to this. This is certainly not what we want as get a null parent/sibling is totally legit case if there is none, operation should not elegantly handle this by checking the nullity of the returned element.